plugins/hls-hlint-plugin/test/Main.hs

```diff
@@ -30,26 +32,40 @@ tests = testGroup "hlint" [
       suggestionsTests
     , configTests
     , ignoreHintTests
+    , applyHintTests
     ]
 
 getIgnoreHintText :: T.Text -> T.Text
 getIgnoreHintText name = "Ignore hint \"" <> name <> "\" in this module"
 
+getApplyHintText :: T.Text -> T.Text
+getApplyHintText name = "Apply hint \"Avoid reverse\""  -- "Apply hint \"" <> name <> "\""
```

Comment:
Commented out code, seems like this is left from debugging?

Reply:
Yeah that was a bad mistake.